data/contributors.json

```diff
@@ -1,35 +1,35 @@
 {
-    "leads": [
```

Comment:
Please run 

`git checkout main data/contributors.json` to restore the original formatting